build.gradle.kts

```diff
@@ -141,4 +153,48 @@ tasks {
       }
     }
   }
+
+  val generateReleaseBundle by registering(Zip::class) {
+    dependsOn(getTasksByName("publishAllPublicationToReleaseRepoRepository", true))
+    from("releaseRepo")
+
+    exclude("**/maven-metadata.*")
+    duplicatesStrategy = DuplicatesStrategy.FAIL
+    includeEmptyDirs = false
+    archiveFileName.set("release-bundle-$stableVersion.zip")
+  }
+
+  val uploadReleaseBundle by registering {
+    dependsOn(generateReleaseBundle)
+
+    val bundle = generateReleaseBundle.get().outputs.files.singleFile
+    val httpClient = OkHttpClient()
+    val username = System.getenv("SONATYPE_USER") ?: throw GradleException("Sonatype user not set")
+    val password = System.getenv("SONATYPE_KEY") ?: throw GradleException("Sonatype key not set")
+    val token = Base64.getEncoder().encodeToString("$username:$password".toByteArray())
+
+    var query = "?name=opentelemetry-java-instrumentation-$stableVersion"
+    // uncomment to automatically publish the release
+    // query += "&publishingType=AUTOMATIC"
```

Comment:
Without this parameter we need to publish the uploaded bundled manually. After first successful release we can uncomment it.